docs/fundamentals/code-analysis/quality-rules/ca2327.md

```diff
@@ -21,12 +21,12 @@ f1_keywords:
 
 ## Cause
 
-This rule fires when both of the following conditions are true for a [Newtonsoft.Json.JsonSerializerSettings](https://www.newtonsoft.com/json/help/html/T_Newtonsoft_Json_JsonSerializerSettings.htm) instanc:
+This rule fires when both of the following conditions are true for a [Newtonsoft.Json.JsonSerializerSettings](https://www.newtonsoft.com/json/help/html/T_Newtonsoft_Json_JsonSerializerSettings.htm) instance:
 
 - The [TypeNameHandling](https://www.newtonsoft.com/json/help/html/P_Newtonsoft_Json_JsonSerializerSettings_TypeNameHandling.htm) property is a value other than `None`.
 - The [SerializationBinder](https://www.newtonsoft.com/json/help/html/P_Newtonsoft_Json_JsonSerializerSettings_SerializationBinder.htm) property is null.
 
-When the [JsonSerializerSettings](https://www.newtonsoft.com/json/help/html/T_Newtonsoft_Json_JsonSerializerSettings.htm) instance is used under these circumstances:
+...and when the [JsonSerializerSettings](https://www.newtonsoft.com/json/help/html/T_Newtonsoft_Json_JsonSerializerSettings.htm) instance is used under these circumstances:
```

Comment:
I haven't seen an ellipsis used this way before. You could make these two bulleted lists into a single list instead.